command/pki_issue_intermediate.go

```diff
@@ -0,0 +1,346 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	paths "path"
+	"strings"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/posener/complete"
+)
+
+type PKIIssueCACommand struct {
+	*BaseCommand
+
+	flagConfig          string
+	flagReturnIndicator string
+	flagDefaultDisabled bool
+	flagList            bool
+
+	flagKeyStorageSource string
+	flagNewIssuerName    string
+}
+
+func (c *PKIIssueCACommand) Synopsis() string {
+	return "Given a Parent Certificate, and a List of Generation Parameters, Creates an Issue on a Specified Moount"
+}
+
+func (c *PKIIssueCACommand) Help() string {
+	helpText := `
+Usage: vault pki issue PARENT CHILD_MOUNT options
+`
```

Comment:
Wanna add the help text like I did with the other command?

Reply:
Uh, here's more help text, hope that it's actually helpful? (Let me know what you thing).

Reply:
This thing, btw: https://github.com/hashicorp/vault/blob/main/command/pki_list_intermediate_command.go#L53-L54

Reply:
Ack, take 2